src/modules/holder/pages/Home/Home.tsx

```diff
@@ -41,20 +50,52 @@ export const Home: FC = () => {
       </>
     )
   }
+
+  // @ts-ignore
+  const validTickets: StoredW3CCredential[] = tickets.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    return Date.parse(credentialSubject?.startDate) >= Date.now()
+  })
+
+  // @ts-ignore
```

Comment:
the same

Reply:
giving this error `Type 'StoredCredential[]' is not assignable to type 'UnsignedW3CCredential[]'.` even though the type is declared as follow

> export type StoredW3CCredential = UnsignedW3CCredential
>
> export type StoredCredential =
  | StoredW3CCredential
  | StoredOpenAttestationDocument
  | (StoredW3CCredential & StoredOpenAttestationDocument)

Reply:
this is because `data` is unknown
```ts
const tickets = data.filter((
```